source/includes/usage-examples/sample-app.kt

```diff
@@ -0,0 +1,25 @@
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.kotlin.client.*
+import com.mongodb.client.model.Filters.*
+
+// Update with the object representation of your collection's documents
+data class Document(val id: String)
```

Comment:
S: If you are parametrizing the collection with the Document class, you do not need to include this data class. It is a generic available in the driver. LMK if you have qs